common/network-common/src/main/java/org/apache/spark/network/util/NettyLogger.java

```diff
@@ -42,6 +45,14 @@ protected String format(ChannelHandlerContext ctx, String eventName, Object arg)
       } else if (arg instanceof ByteBufHolder) {
         return format(ctx, eventName) + " " +
           ((ByteBufHolder) arg).content().readableBytes() + "B";
+      } else if (arg instanceof InputStream) {
+        int available = 0;
+        try {
+          available = ((InputStream) arg).available();
+        } catch (IOException ex) {
+          // Swallow
+        }
+        return format(ctx, eventName) + " " + available + "B";
```

Comment:
Specific input streams might have additional details that can be useful.

```suggestion
        return format(ctx, eventName, arg) + " " + available + "B";
```